src/database/DataTypes/LocationType.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2020
+ */
+
+import Realm from 'realm';
+
+export class LocationType extends Realm.Object {}
+
+LocationType.schema = {
+  name: 'LocationType',
+  primaryKey: 'id',
+  properties: {
+    id: 'string',
+    description: { type: 'string', optional: true },
+    comment: { type: 'string', optional: true },
+  },
+};
+
+export default LocationType;
```

Comment:
maybe this can not be a class with it's own file but I like it much better like that!

Reply:
I like the separate file. touch and go with this one because it's small at this stage, but go for it anyway